paasta_tools/tron_tools.py

```diff
@@ -718,10 +718,15 @@ def format_tron_action_dict(action_config: TronActionConfig, use_k8s: bool = Fal
     # whatever is in soaconfigs to the k8s equivalent here as well.
     if executor in KUBERNETES_EXECUTOR_NAMES and use_k8s:
         result["executor"] = "kubernetes"
+
         result["secret_env"] = action_config.get_secret_env()
-        # For k8s, we do not want secret envvars to be duplicated in both `env` and `secret_env`
         all_env = action_config.get_env()
+        # For k8s, we do not want secret envvars to be duplicated in both `env` and `secret_env`
         result["env"] = {k: v for k, v in all_env.items() if not is_secret_ref(v)}
+        # for Tron-on-K8s, we want to ship tronjob output through logspout
+        # such that this output eventually makes it into our per-instance
+        # log streams automatically
+        result["env"]["TRON_ENABLE_LOGSPOUT"] = "1"
```

Comment:
Sorry, thinking about this also -- technically this toggle controls whether we send to a per-instance stream_paasta_* stream vs per-service, logspout is already 'enabled' on tron.  

Seems like a very minor nit if this is going to be enabled everywhere eventually, so feel free to not block on this.

Reply:
`@marley what is temporary`

nah, you're right - I can rename this to `TRON_ENABLE_PER_INSTANCE_LOGSPOUT` just to be more explicit